numpyro/infer/elbo.py

```diff
@@ -311,53 +312,115 @@ class RenyiELBO(ELBO):
         Here :math:`\alpha \neq 1`. Default is 0.
     :param num_particles: The number of particles/samples
         used to form the objective (gradient) estimator. Default is 2.
+    :param callable create_plates: An optional function inputing the same
```

Comment:
can we add a usage example to the doc string?

Reply:
done